src/Gutenberg.php

```diff
@@ -29,6 +32,26 @@ public function load_products() {
 
 	public function load_block_assets() { // phpcs:ignore
 
+	    $build_dir_url  = $this->pluginDirUrl . 'build/';
+	    $build_dir_path = $this->pluginDirPath . 'build/';
+
+	    if (file_exists($build_dir_path . 'ss-gutenberg-test.asset.php')) {
```

Comment:
Po buildu je vygenerován PHP soubor, který obsahuje závislosti a verzi scriptu. To následně využijeme pro `wp_register_script`.